tcms/core/middleware.py

```diff
@@ -29,6 +30,26 @@ def process_request(self, request):
                       '<a href="%(admin_url)s">change it</a>') % {
                           'doc_url': doc_url,
                           'admin_url': reverse('admin:sites_site_change', args=[site.pk])
-                      }
+                          }
+                )
+            )
```

Comment:
these also seem unrelated. Why do we need them ?

Reply:
I had noticed that pylint complained about `Wrong hanging indentation.pylint(bad-continuation)`
I had been trying to ignore the error because my editor formats the code on saving (pylint). I unintentionally added it on the last push, will remove.